docs/BR.md

```diff
@@ -1808,20 +1810,32 @@ If the Subordinate CA is not allowed to issue certificates with dNSNames, then t
+If the Server Certificate asserts the policy identifier of 2.23.140.1.2.1, then it MUST NOT include organizationName, givenName, surname, streetAddress, localityName, stateOrProvinceName, or postalCode in the Subject field.
 
-{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1) baseline-requirements(2) individual-validated(3)} (2.23.140.1.2.3), if the Certificate complies with these Requirements and includes Subject Identity Information that is verified in accordance with Section 3.2.3.
+{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1) baseline-requirements(2) organization-validated(2)} (2.23.140.1.2.2), if the Server Certificate complies with these Requirements and includes Subject Identity Information that is verified in accordance with Section 3.2.2.1.
 
-If the Certificate asserts the policy identifier of 2.23.140.1.2.2, then it MUST also include organizationName, localityName (to the extent such field is required under Section 7.1.4.2.2), stateOrProvinceName (to the extent such field is required under Section 7.1.4.2.2), and countryName in the Subject field.  If the Certificate asserts the policy identifier of 2.23.140.1.2.3, then it MUST also include (i) either organizationName or givenName and surname, (ii) localityName (to the extent such field is required under Section 7.1.4.2.2), (iii) stateOrProvinceName (to the extent required under Section 7.1.4.2.2), and (iv) countryName in the Subject field.
+If the Server Certificate asserts the policy identifier of 2.23.140.1.2.2, then it MUST also include organizationName, localityName (to the extent such field is required under Section 7.1.4.2.2), stateOrProvinceName (to the extent such field is required under Section 7.1.4.2.2), and countryName in the Subject field. 
+
+{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1) baseline-requirements(2) individual-validated(3)} (2.23.140.1.2.3), if the Server Certificate complies with these Requirements and includes Subject Identity Information that is verified in accordance with Section 3.2.3.
+
+If the Server Certificate asserts the policy identifier of 2.23.140.1.2.3, then it MUST also include (i) either organizationName or givenName and surname, (ii) localityName (to the extent such field is required under Section 7.1.4.2.2), (iii) stateOrProvinceName (to the extent required under Section 7.1.4.2.2), and (iv) countryName in the Subject field.
+
+{joint‐iso‐itu‐t(2) international‐organizations(23) ca‐browser‐forum(140) certificate‐policies(1) ev-guidelines (1) }
+(2.23.140.1.1), if the Server Certificate complies with these Requirements and has been issued and operated in accordance with the CA/Browser Forum Guidelines for the Issuance and Management of Extended Validation Certificates ("EV Guidelines"), in addition to these Requirements.
```

Comment:
I'm also interested in Corey's wise counsel on this, but the approach seems reasonable to me.

Reply:
I think that approach sounds good, although I imagine we probably want to further restrict the allowed set of OIDs. Specifically:
1.    2.23.140.1.1 OID (EV)
 2.   2.23.140.1.2 arc (BR) or exhaustively list the OIDs underneath this arc

The rationale being that there has been confusion in the past (https://groups.google.com/d/msg/mozilla.dev.security.policy/Q2k_5eGXqmA/CHDrlZYPDgAJ) regarding proper use of CABF OIDs, so being more specific will provide guidance and avoid potentially messy situations moving forward. Additionally, it looks like the cabf-TorServiceDescriptor OID was incorrectly added to the "CABF Policy OID" arc (but it's actually to be used as an extension OID), so we definitely want to avoid any confusion there.